src/components/ImageView/index.native.js

```diff
@@ -66,12 +66,18 @@ class ImageView extends PureComponent {
         if (!this.props.url) {
             return;
         }
-        ImageSize.getSize(this.props.url).then(({width, height}) => {
+        ImageSize.getSize(this.props.url).then(({width, height, rotation}) => {
             let imageWidth = width;
             let imageHeight = height;
             const containerWidth = Math.round(this.props.windowWidth);
             const containerHeight = Math.round(this.state.containerHeight);
 
+            // On specific Android devices, the dimensions are sometimes returned to us flipped here, with `rotation` set to 90 degrees.
+            // Swap them back to make sure the image fits nicely in the container. On iOS, the rotation is always undefined, so this does not apply.
+            if ((rotation === 90 || rotation === 270)) {
```

Comment:
```suggestion
            if (rotation === 90 || rotation === 270) {
```

Remove the extra set of parenthesis and we are done :)

Reply:
Oh man how did I not see this :|